### DISCUSSION TITLE
Missing GitHub Foundations Exam Completion Email

### DISCUSSION THREAD

**Discussion by ankitrai1985** [community/community#134339] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hello Community :) 

I passed the GitHub Foundations exam on the morning of July 29th, but I haven't received a passing email or Credly badge yet. I'm unsure if I should wait longer or seek assistance. Could you please guide me on how to proceed?

Thank you,
Ankit

**Comment by queenofcorgis** [comment]
@ankitrai1985 I'd love to help here, I have a few questions to get us started. Have you checked your spam folder? And can you please confirm you're still experiencing this?

**Comment by ankitrai1985** [comment]
Hello @queenofcorgis , Thank you very much for offering your help. I still haven't received the exam pass email or the Credly badge. I have also raised a support ticket, which you can find at [this link](https://support.github.com/ticket/personal/0/2914730), in case you have access to it.

**Comment by yuriachermann** [comment]
@ankitrai1985 I'm having the same issue... Could you solve that?
@queenofcorgis any updates?

**Comment by queenofcorgis** [comment]
Please email [gh-certification@github.com](mailto:gh-certification@github.com) to assist with any missing emails or badges. Thank you!

**Comment by subhagitpom** [comment]
I am having the same issue

**Reply by Saikrishna1504** [reply]
Same here

**Reply by subhagitpom** [reply]
Same here for copilot exam

**Reply by Saikrishna1504** [reply]
Letme know if you find any solution please also mail gh-certification@github.com for better response

**Reply by divyansh122** [reply]
Is your problem solved ?????

**Reply by Ramasai88** [reply]
Same Problem 
did anyone got the certificate

**Comment by subhagitpom** [comment]
Nope

**Reply by divyansh122** [reply]
I also raised a query but no updates . I had given the foundations exam

**Comment by Manthanchiddarwar** [comment]
I have also same problem didn't receive yet the batch and certificate

**Reply by divyansh122** [reply]
![Screenshot_2025-02-17-08-14-02-33_40deb401b9ffe8e1df2f1cc5ba480b12.jpg](https://github.com/user-attachments/assets/ce92ed91-979f-4642-a16d-5e4cb918391b)

Today morning I was shown this on the github certifications page ! they are resolving this problem

**Reply by Saikrishna1504** [reply]
I got mail from them just now , it says that they are aware of psi related misisng badges & score issue will be resolved within a week .

**Reply by Manthanchiddarwar** [reply]
Sure sir waiting for Batches & Certificate and if in case not get it will contact you again !

**Comment by divyansh122** [comment]
did any one got the certificate  ?

**Reply by Saikrishna1504** [reply]
not yet

**Reply by sammorris66** [reply]
No, I'm still waiting for the certificate, but PSI resent me the confirmation that I passed the exam earlier today.

**Comment by Manthanchiddarwar** [comment]
When we will get ?

**Reply by Saikrishna1504** [reply]
All we can do is wait until Monday...if it's still same then we can resend mail

**Reply by Manthanchiddarwar** [reply]
Any further update?

**Reply by divyansh122** [reply]
Not recieved yet !

**Reply by meIwata** [reply]
I took the exam on February 19th. I received an email from Credly with my earning badge first, followed by an email from PSI Exams confirming the completion of the exam and my passing score. I hope this helps.
![image](https://github.com/user-attachments/assets/e463782f-0d21-4551-b7c7-dacdf8d8c773)
![image](https://github.com/user-attachments/assets/9813db22-2ba1-40db-baeb-1b7b7e2c271f)

**Reply by Saikrishna1504** [reply]
I completed my exam on 16th February still I haven't received any mail or badge 😔

**Reply by Saikrishna1504** [reply]
Everyone check your mails , certificates/badge has been sent along with your scores 😄

**Reply by Manthanchiddarwar** [reply]
Received!!
